src/index.d.ts

```diff
@@ -37,7 +37,7 @@ declare module 'minecraft-protocol' {
 		writeChannel(channel: any, params: any): void
 		signMessage(message: string, timestamp: BigInt, salt?: number): Buffer
 		verifyMessage(publicKey: Buffer | KeyObject, packet: object): boolean
-		reportPlayer(uuid: string, reason: 'FALSE_REPORTING' | 'HATE_SPEECH' | 'TERRORISM_OR_VIOLENT_EXTREMISM' | 'CHILD_SEXUAL_EXPLOITATION_OR_ABUSE' | 'IMMINENT_HARM' | 'NON_CONSENSUAL_INTIMATE_IMAGERY' | 'HARASSMENT_OR_BULLYING' | 'DEFAMATION_IMPERSONATION_FALSE_INFORMATION' | 'SELF_HARM_OR_SUICIDE' | 'ALCOHOL_TOBACCO_DRUGS', signatures: Buffer[], comment?: string)
+		reportPlayer(uuid: string, reason: 'FALSE_REPORTING' | 'HATE_SPEECH' | 'TERRORISM_OR_VIOLENT_EXTREMISM' | 'CHILD_SEXUAL_EXPLOITATION_OR_ABUSE' | 'IMMINENT_HARM' | 'NON_CONSENSUAL_INTIMATE_IMAGERY' | 'HARASSMENT_OR_BULLYING' | 'DEFAMATION_IMPERSONATION_FALSE_INFORMATION' | 'SELF_HARM_OR_SUICIDE' | 'ALCOHOL_TOBACCO_DRUGS', signatures: Buffer[], comment?: string): true
```

Comment:
reportPlayer returns Promise<true>

Reply:
Thanks, didn't think about that. Fixed in https://github.com/PrismarineJS/node-minecraft-protocol/compare/61dd6a9eb7d8d2082f55196076addebcc25e5dbf..528a0fa1fff2ff6d8f1bd6c7d7bd1e6a25a0183a